src/lib/statsv2.svelte

```diff
@@ -0,0 +1,179 @@
+      picture: 'statsv2/statImage6.png'
+    }
+  ];
+
+  let screenWidth: number;
+
+  const updateScreenWidth = () => {
+    screenWidth = window.innerWidth;
+  };
+
+  onMount(() => {
+    if (typeof window !== 'undefined') {
+      window.addEventListener('resize', updateScreenWidth);
+      updateScreenWidth();
+    }
+  });
+
+  onDestroy(() => {
+    if (typeof window !== 'undefined') {
+      window.removeEventListener('resize', updateScreenWidth);
+    }
+  });
+</script>
+
+<h1>
+  FixNation’s mission is to reduce the population of homeless cats by sterilizing as many as
+  possible while demonstrating the effectiveness of Trap-Neuter-Return and colony management for the
+  humane care of homeless cats.
+</h1>
+<div>
+  {#if screenWidth > 675}
+    <div class="slider">
+      {#each sections as section}
+        <section class="section">
+          <img src={section.picture} />
+          <span class="rectangle" />
+          <div class="text">
+            <div class="header">{section.number}</div>
+            <div class="subtitle">{section.subtitle}</div>
+          </div>
+        </section>
+      {/each}
+    </div>
+  {:else}{/if}
+</div>
+
+<style>
+  h1 {
+    font-family: 'Avenir', sans-serif !important;
+    font-style: normal;
+    font-weight: 500;
+    text-align: center;
+    font-size: 1rem;
+    margin: 1vw;
+    padding: 2rem;
+  }
+  * {
+    box-sizing: border-box;
+    margin: 0;
+    padding: 1;
+  }
+
+  section {
+    padding: 1rem;
+    min-width: 23rem;
+    height: auto;
+    scroll-snap-align: start;
+    text-align: center;
+    position: relative;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+  }
+  img {
+    width: 20rem;
+    height: auto;
+    margin-left: auto;
+    margin-right: auto;
+  }
+  .header {
+    color: #ffffff;
+    font-weight: 700;
+    letter-spacing: 0.02rem;
+    font-size: 4rem;
+    font-family: 'Rubik';
+  }
+  .subtitle {
+    color: #ffffff;
+    font-size: 1.3rem;
+    font-family: 'Avenir', sans-serif !important;
+    font-style: normal;
+    font-weight: 40;
+  }
+  .rectangle {
+    position: absolute;
+    width: 14rem;
+    height: 14rem;
+    margin: 1rem;
+    background: #008e7e;
+    opacity: 0.75;
+    box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
+    border-radius: 4px;
+  }
+  .text {
+    width: 13.5rem;
+    height: 13.5rem;
+    position: absolute;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+  }
+
+  .slider {
+    scroll-snap-type: x mandatory;
+    display: flex;
+    -webkit-overflow-scrolling: touch;
+    overflow-x: scroll;
+  }
+  ::-webkit-scrollbar {
+    width: 0px;
+    background: transparent; /* make scrollbar transparent */
+  }
+
+  /* Mobile variant only */
+  @media screen and (max-width: 675px) {
+    .rectangle {
+      width: 60vw;
+      height: 60vw;
+    }
+    .text {
+      width: 60vw;
+      height: 60vw;
```

Comment:
Is it intentional that you have the height in view width units (vw)? I'm just wondering since below this, you use view height units (vh) for the image height.

Reply:
I know you only did the web view so far, but if the mobile view is going to use this code later on, I wanted to address this to avoid any unintentional styling typos.